src/anbox/utils.cpp

```diff
@@ -193,5 +194,17 @@ bool is_mounted(const std::string &path) {
 
 }
 
+std::string find_program_on_path(const std::string &name) {
```

Comment:
`get_full_path` is misleading as it suggests just that it expands the given thing to a full path but actually this function searches for the right path. So `find_` as prefix is correct. Using `bool` would contradict the `find_` prefix. I initially added this function to then use the returned path as input for `exec` but stopped doing that in the final implementation. Lets leave this as is as a nice helper with a possible future use case.